gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/process/distributed/mapper/AccessPointMapper.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.gateway.services.sync.process.distributed.mapper;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.gravitee.gateway.services.sync.process.repository.synchronizer.accesspoint.AccessPointDeployable;
+import io.gravitee.repository.distributedsync.model.DistributedEvent;
+import io.gravitee.repository.distributedsync.model.DistributedEventType;
+import io.gravitee.repository.management.model.AccessPoint;
+import io.reactivex.rxjava3.core.Maybe;
+import java.util.Date;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@RequiredArgsConstructor
+@Slf4j
+public class AccessPointMapper {
+
+    private final ObjectMapper objectMapper;
+
+    public Maybe<AccessPointDeployable> to(final DistributedEvent event) {
+        return Maybe.fromCallable(() -> {
+            try {
+                final AccessPoint accessPoint = objectMapper.readValue(event.getPayload(), AccessPoint.class);
+                return AccessPointDeployable
+                    .builder()
+                    .accessPoint(accessPoint)
```

Comment:
You should create a gateway version of this model, as explained above and no refer to repository model.